app/assets/stylesheets/master.scss

```diff
@@ -138,8 +138,12 @@ body {
     width: 11vh;
     padding-bottom: 12.5%;
 }
+.cell-highlight {
+  outline-offset: -5px;
+  outline: 5px solid red;
+}
 
-#board td:hover {
+.hover-cell:hover {
   outline-offset: -5px;
   outline: 5px solid #94eded;
 }
```

Comment:
I think this should stay the generic board td: hover, otherwise it will only work on occupied cells and the ui should have the generic hover feedback for all cells.